packages/2d/src/components/Line.ts

```diff
@@ -199,6 +193,10 @@ export class Line extends Shape {
       }
     }
 
+    if (this.closed()) {
```

Comment:
I just figured out that this won't work like this as it it's going to _always_ close the path even though that segment should not even be shown (because of `start`, `end`, or the offset). I'll have to figure something out there.

Reply:
Nevermind, I just need to add `this.end() === 1` to the condition.